common/components/DashboardPage/blocks/categories.tsx

```diff
@@ -20,11 +21,6 @@ const CategoriesBlock: React.FC<{ style: string }> = ({ style }) => {
   const { data: categoriesData } = useGetAllCategoriesQuery('')
   const categories = categoriesData?.data
 
-  const getCount = (name) => {
-    return tasks?.filter((task) => task?.category == name)
-  }
-  // console.log(getCount('Vodniki1'))
-
   const columns = [
```

Comment:
You should eject constants to another file the in next cases:
- const initialization takes up a lot of space
- you have many constants in the file

If this constant don't use anywhere in the code, then create a new file (`categories.config.ts`) near the current and move the constant to it.